### DISCUSSION TITLE
Suddenly Vercel is no longer updating my deployments from  Github.

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#6534] [Account Issue]
### Question

It worked before. I changed nothing in the connection.
Why?
This is my account: https://vercel.com/ingedingens-projects/nicethings
There is NO error message. 
I am on a hobby account, could that be the reason? If so, why does not anyone say so?

**Comment by versecafe** [comment]
@ingedingen Is it still tracking an attempt to make the deployment on githubs side?

**Reply by ghost** [reply]
I am sorry but I do not know what that question means.

**Comment by RinKimLaw** [comment]
Try doing a manual deployment to see if deploys are still happening.

One potential issue could be that you're making commits to the repo with a different email address than the email address linked to your Vercel account (Or if you're in a Vercel team, then your commits are authored by an email address that's not on the team).

**Reply by ghost** [reply]
I am sorry, but I do not know how.  But there is no change in email address.
